src/main/java/org/apache/maven/wrapper/Logger.java

```diff
@@ -0,0 +1,18 @@
+package org.apache.maven.wrapper;
+
+/**
+ * @author <a href="mailto:konstantin.sobolev@gmail.com">Konstantin Sobolev</a>
+ */
+public class Logger {
+  private static final boolean VERBOSE = "true".equalsIgnoreCase(MavenWrapperMain.MVNW_VERBOSE);
```

Comment:
Hey, 
I'm not completely agree with the suggestion, but just looking the PR, this line seems wrong.
It is referencing a string.
Either here or `MavenWrapperMain.MVNW_VEREBOSE` needs to check the value from system env.

Reply:
Oops, my bad, you're right. Updated.